src/Microsoft.ML.OnnxConverter/OnnxContextImpl.cs

```diff
@@ -313,6 +313,17 @@ public override string AddInitializer(IEnumerable<long> values, IEnumerable<long
             return name;
         }
 
+        public override string AddInitializer(IEnumerable<double> values, IEnumerable<long> dims, string name = null, bool makeUniqueName = true)
+        {
```

Comment:
Is the double version of AddInitializer being used anywhere? I couldnt see it being used in any obvious manner in the onnx export function.

Reply:
I use it in line 414 of the export function.